resolwe_runtime_utils.py

```diff
@@ -275,12 +288,177 @@ def checkrc(rc, *args):
     return json.dumps(ret)
 
 
+CHUNK_SIZE = 10000000  # 10 Mbytes
+
+
+class ImportedFormat:
+    """Import destination file format."""
+
+    EXTRACTED = 'extracted'
+    COMPRESSED = 'compressed'
+    BOTH = 'both'
+
+
+def import_file(src, file_name, imported_format=ImportedFormat.BOTH, max_progress=None):
+    """Import file to working directory.
+
+    :param src: Source file path or URL
+    :param file_name: Source file name
+    :param imported_format: Import file format (extracted, compressed or both)
+    :param max_progress: Progress value after import
+    :return: Destination file path (if extracted and compressed, extracted path given)
+    """
+    print("Importing and compressing {}...".format(file_name))
+
```

Comment:
Given how some tests fail now, consider adding a check prior to extraction that the input file actually exists. It'll be easier to debug if we have a dedicated error message so that we don't need to dig into the output of `subprocess.check_call`.